data/heartbeat/proto/heartbeat.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package proto;
+
+option go_package = "heartbeat";
+
+// Heartbeat represents the heartbeat message that is sent between peers from the same shard containing
+// current node status
+message Heartbeat {
+    bytes   Payload         = 1;
+    uint32  ShardID         = 2;
+    string  VersionNumber   = 3;
+    string  NodeDisplayName = 4;
```

Comment:
do we have validation for all these fields for maximum length ?

Reply:
yes, basically the same implementation (copied because the old heartbeat package will simply disappear) in process/heartbeat/interceptedPeerHeartbeat.go CheckValidity function.